ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -342,6 +356,240 @@ objects:
           oc wait namespace/openshift-ssh-bastion --for delete --timeout=10m
         }
 
+        function recover-from-etcd-quorum-loss() {
+          mkdir -p ~/.ssh || true
+          cp "${KUBE_SSH_KEY_PATH}" ~/.ssh/id_rsa
+          chmod 0600 ~/.ssh/id_rsa
+          echo "${USER:-default}:x:$(id -u):$(id -g):Default User:$HOME:/sbin/nologin" >> /etc/passwd
+
+          echo "Setting up ssh bastion"
+          curl https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh | bash
```

Comment:
It would be nice to share code instead of duplicating [this existing logic][1] here.  But I guess this is all going to go away a few minutes after this PR lands? ;)

[1]: https://github.com/openshift/release/blob/6ca4a9371305b26407c90c2fe82a98c83b93591a/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L197

Reply:
Fixed in 1484b5006